src/api.ts

```diff
@@ -26,3 +26,13 @@
 export const getTodos = () => get<Todo[]>('/todos');
 
 export const getUser = (userId: number) => get<User>(`/users/${userId}`);
+
+export const getComplited = () => {
```

Comment:
Consider renaming the function to follow the naming conventions for methods. A more descriptive name like 'fetchCompletedTodos' would be clearer.